### DISCUSSION TITLE
octokit.rest.teams.create seems to ignore parent_team_id parameter

### DISCUSSION THREAD

**Discussion by gigamonkey** [octokit/.github#50] [Q&A] [answered]
Hi, I'm lost in a twisty maze of github repos, all slightly different so I'm not sure where's the best place to ask this or file an issue. But anyway, I'm using Octokit with a GitHub app and when I try to create a team with the `octokit.rest.teams.create` method in the organization it works fine but totally ignores the `parent_team_id` parameter. I can create the team  and then update it with `octokit.rest.teams.updateInOrg` and set the parent that way but I'm wondering if I'm doing something else wrong. My call to create looks like:

```javascript
octokit.rest.teams.create({
      org: this.org,
      name: name,
      description: description,
      privacy: 'closed',
      parent_team_id: parent_team_id,
});
```

If there's a better place to ask about this, please point me in the right direction.

**Comment by gr2m** [comment, marked as answer]
can you try the following please

1. Run the code you shared but with an `.endpoint` suffix and log out the request parameters
2. Try to reproduce the problem with the given request options using `curl` or another tool to send requests

```js
const requestOptions = octokit.rest.teams.create.endpoint({
      org: this.org,
      name: name,
      description: description,
      privacy: 'closed',
      parent_team_id: parent_team_id,
});

console.log(requestOptions)
```

I think it's most likely a bug in your code (`parent_team_id` is undefined or invalid) or in GitHub's API.

**Comment by gigamonkey** [comment]
I'm sad to say you're probably right that it must have been a bug in my code. Because in the course of making a stripped down program to demonstrate the endpoint thing I got to something that works. Unfortunately I didn't commit all my previous non-working versions so I can't figure out what I did wrong before. Anyway, thanks for the answer. And that `.endpoint` trick is cool so at least I learned something. Cheers!
